doc/visual-programming/source/widgets/visualize/linearprojection.rst

```diff
@@ -37,20 +57,31 @@ virginica*.
 .. figure:: images/linear-projection-stamped.png
 
 1. Axes in the projection that are displayed and other available axes.
-2. Set the color of the displayed dots (you will get colored dots for
-   discrete values and grey-scale dots for continuous). Set opacity,
-   shape and size to differentiate between instances.
-3. Set `jittering <https://en.wikipedia.org/wiki/Jitter>`_ to prevent
+2. Optimize your projection by using **Suggest Features**. This feature
+   scores attributes by average classification accuracy and returns the
+   top scoring attributes with a simultaneous visualization update.
+3. Choose the type of projection.
```

Comment:
Indenting is all off. Please fix. Never forget to check how the docs look like when built with `make html`. Sphinx will report errors in docs. You also need to `pip install CommonMark==0.5.5`. Weird stuff. 🤷‍♀️